extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.ts

```diff
@@ -149,7 +150,8 @@ export class RatioExpressionInputValidationService {
               });
             }
           } else if (currentRuleType === 'HasSpecificTermEqualTo') {
-            let termIndex = currentInput[0]; // Note: termIndex is 1-indexed.
+            const _currentInput = <number[]> currentInput;
+            let termIndex = _currentInput[0]; // Note: termIndex is 1-indexed.
```

Comment:
The note was actually made in PR [#12103](https://github.com/oppia/oppia/pull/12103)
```
...rule input to be 1-indexed, not 0-indexed. In other words, we don't want the lesson 
implementor to have a rule like "make sure the 0-th term of the ratio equals..." since
the 0-th term doesn't exist. 
```

Reply:
Can you add the whole comment in the code? (I just realized you didn't add this but it will make this easier to read).